composer.json

```diff
@@ -29,30 +29,30 @@
         "php": "^7.2",
         "ext-ctype": "*",
         "ext-iconv": "*",
-        "dantleech/phpcr-migrations-bundle": "^1.0",
-        "doctrine/doctrine-bundle": "^1.10 || ^2.0",
+        "dantleech/phpcr-migrations-bundle": "^1.2",
+        "doctrine/doctrine-bundle": "^2.0",
```

Comment:
As discussed with @chirimoya as the recipes only match the new version we will require the newest symfony package versions in the skeleton for new projects.

Reply:
This is not really a BC break, because existing projects don't necessarily have to update, right?

Reply:
Yes this is not a BC Break, the skeleton should from my side always represent how a new project should look like.